driver/src/test/scala/org/mongodb/scala/FindObservableSpec.scala

```diff
@@ -57,6 +58,7 @@ class FindObservableSpec extends FlatSpec with Matchers with MockFactory {
     wrapper.expects('first)(*).once()
     wrapper.expects('filter)(filter).once()
     wrapper.expects('maxTime)(duration.toMillis, TimeUnit.MILLISECONDS).once()
+    wrapper.expects('maxAwaitTime)(duration.toMillis, TimeUnit.MILLISECONDS).once()
```

Comment:
Just to ensure that maxTime and maxAwaitTime are not mixed up, it would be safer to use a different duration for each.